components/profile/Authenticate.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {signin, useSession, signout} from 'next-auth/client';
+
+const Authenticate = () => {
```

Comment:
Can you make a branch from github repo and add this logic? I can't checkout to the branch to test out your feature